popup-wrapper.js

```diff
@@ -1,10 +1,23 @@
 chrome.tabs.query({active: true, currentWindow: true}, (tabs) => {
-    chrome.tabs.sendMessage(tabs[0].id, {
-        type: "popup"
-    },
-    (response) => {
-        if (response) {
-            //console.log(response);
+    if (tabs[0] === undefined || !tabs[0].url.startsWith('https://www.youtube.com/')) {
+        const errorMessageSpan = document.querySelector('#extErrorMessage')
+        errorMessageSpan.textContent = 'You need to be on YouTube!'
+        return
+    }
+
+    chrome.tabs.executeScript(tabs[0].id, {
+        // showPopup acutally toggles popup here
+        code: 'window.__PREVYOU_LOADED ? (showPopup(), true) : false'
```

Comment:
`executeScript` will pass to the callback the return value of the last executed statement in the injected script.

If not in a one-liner, the following code would do the same thing:

```js
if (window.__PREVYOU_LOADED) {
  showPopup()
  return true
} else {
  return false
}
```

This allows in the same expression to toggle the popup if PrevYou is already loaded, and still return `true` to the calling script